src/overlays/actors/ovl_Dm_Char00/z_dm_char00.c

```diff
@@ -1066,12 +1066,13 @@ void DmChar00_Draw(Actor* thisx, PlayState* play2) {
 
     Gfx_SetupDL27_Xlu(play->state.gfxCtx);
 
-    do {
-        phi_a0 = (this->unk_262 * 50) & 511;
-        if (phi_a0 >= 256) {
-            phi_a0 = 511 - phi_a0;
-        }
-    } while (0);
+    //! FAKE:
+    if (1) {}
+
+    phi_a0 = (this->unk_262 * 50) & 511;
```

Comment:
Since you are cleaning this up, this way of writing it makes a bit more of sense given the context
```suggestion
    phi_a0 = (this->unk_262 * 50) % 512;
```

Reply:
It needs to be unsigned for this to work. Is `(this->unk_262 * 50U) % 512;` fine?

Reply:
I think either `(this->unk_262 * 50U) % 512;` or `(this->unk_262 * 50) % 512U;` are fine. I prefer the latter but either is fine to me

Reply:
ftr I like the latter as well.